azkaban-common/src/main/java/azkaban/imagemgmt/models/ImageRampRule.java

```diff
@@ -0,0 +1,50 @@
+package azkaban.imagemgmt.models;
+
+import java.util.List;
+import java.util.Set;
+
+
+public class ImageRampRule extends BaseModel {
+  // ramp rule name
+  String ruleId;
```

Comment:
Minor: can we rename the ruleId into ruleName? It does not look correct to me b/c I think *Id should be an integer?

Reply:
+1

Reply:
done